_posts/2019-10-08-unit-testing-basics.md

```diff
@@ -1138,9 +1138,39 @@ tests, so you can be sure that the piece of code was tested in some capacity.
 Note that code coverage doesn't measure the different behaviors a code path
 **could take**, but only that a particular code path **was taken**.
 
-![]({% img_url unit-testing-basics/code_coverage.png %})
+```
+$ sudo apt install lcov
+$ make lcov
+make -f minimal/tests/makefiles/Makefile_sum.mk
+make[1]: Entering directory 'minimal/tests'
+Running build/sum/sum_tests
+.
+OK (1 tests, 1 ran, 1 checks, 0 ignored, 0 filtered out, 0 ms)
+
+make[1]: Leaving directory 'minimal/tests'
+lcov --base-directory . --directory . -c -o build/lcov.info --exclude "*cpputest/*" --exclude "*tests/*"
 
-Above is a an example coverage report taken from the Memfault Public SDK[^5].
+...
+
+genhtml -o test_coverage -t "coverage" --num-spaces 4 build/lcov.info -o build/test_coverage/
+Reading data file build/lcov.info
+Found 1 entries.
+Writing .css and .png files.
+Generating output.
+Processing file minimal/my_sum.c
+Writing directory view page.
```

Comment:
could strip out a few more of these lines to make this snippet less verbose

Reply:
Agreed, proposing the change